clients/java/src/main/java/io/camunda/zeebe/client/api/command/FinalCommandStep.java

```diff
@@ -15,32 +15,36 @@
  */
 package io.camunda.zeebe.client.api.command;
 
+import io.camunda.zeebe.client.api.CamundaFuture;
 import io.camunda.zeebe.client.api.ZeebeFuture;
 import java.time.Duration;
 
 public interface FinalCommandStep<T> {
 
   /**
    * Sets the request timeout for the command. The default request timeout can be configured using
-   * {@link io.camunda.zeebe.client.ZeebeClientBuilder#defaultRequestTimeout(Duration)}.
+   * {@link io.camunda.zeebe.client.CamundaClientBuilder#defaultRequestTimeout(Duration)}.
    *
    * @param requestTimeout the request timeout
    * @return the configured command
    */
   FinalCommandStep<T> requestTimeout(Duration requestTimeout);
 
   /**
-   * Sends the command to the Zeebe broker. This operation is asynchronous. In case of success, the
-   * future returns the event that was generated by the Zeebe broker in response to the command.
-   *
-   * <p>Call {@link ZeebeFuture#join()} to wait until the response is available.
-   *
-   * <pre>
+   * @deprecated Use {@link #send()} instead
+   *     <p>Sends the command to the Zeebe broker. This operation is asynchronous. In case of
+   *     success, the future returns the event that was generated by the Zeebe broker in response to
+   *     the command.
+   *     <p>Call {@link CamundaFuture#join()} to wait until the response is available.
+   *     <pre>
    * Future&#60;JobEvent&#62 future = command.send();
    * JobEvent event = future.join();
    * </pre>
    *
    * @return a future tracking state of success/failure of the command.
    */
+  @Deprecated
   ZeebeFuture<T> send();
+
+  CamundaFuture<T> sendCommand();
```

Comment:
❓ This will mean a lot of code changes for users, I guess? Do we really need this?

Reply:
I added this method to not make users change their code. Because I can't change/override the return type of the `send` method

Reply:
Oh, yeah, I see. Well, that's a bit tricky 🤔 
I'll try to find some time to think about it again.

Reply:
Thinking about this again, I would still like to avoid introducing this new method.
What happens if we return a `CamundaFuture` from the `send` method here and the `CamundaFuture` extends the `ZeebeFuture`? 
If I currently have a variable `final ZeebeFuture result`, the send method should still work, shouldn't it? I now receive a `CamundaFuture` but that is just a specialized `ZeebeFuture`, it still fits into the `ZeebeFuture` and my code should still be fine. I'll still only have access to the `ZeebeFuture`-part of the return `CamundaFuture` but that should be OK.

Reply:
It's not doable. `CamundaFuture` extends `ZeebeFuture`, if we change the return type also users might need to do that

Reply:
We would have to adjust the command implementations to already return the `CamundaFuture`s as well. But consumers of the commands would be fine, they could keep their `ZeebeFuture` variables or move to the `CamundaFuture` type, as they like.

Reply:
I can still do this, with 8.5 and 8.6:
```java
final ZeebeFuture<DeleteResourceResponse> future = client.newDeleteResourceCommand(123).send();
future.join();
```

Reply:
Let me try

Reply:
✅

Reply:
Seems not working as expected

Reply:
Oh because tasklist is using mockito on the return type, let me fix that

Reply:
OK, I see the issue. It is a breaking change for sure since we're adjusting the return type of the method.
Well, I guess, if we want to stay backward-compatible, we'll have to introduce a new method 😞 
Sorry for the noise and additional work. We can also represent it on Monday again in the team and ask for feedback but I think there aren't many ways around it...

Reply:
I will rollback the changes, and raise it to the team